aemanalyser-maven-plugin/src/main/java/com/adobe/aem/analyser/mojos/AemAnalyseMojo.java

```diff
@@ -126,6 +128,13 @@
     @Parameter(property = "aem.analyser.classifier")
     private String classifier;
 
+    /**
+     * Analyzes the given list of content package files.
+     * If this is configured, only these files are validated, and not the main project artifact or dependencies. 
+     */
+    @Parameter
+    private List<File> contentPackageFiles;
```

Comment:
Do we need to use "File" here? Are those files usually part of the project source or in the target directory?
I want to avoid that the files can be anywhere in the file system, outside the project boundary

Reply:
our use case are "all" content package files generated by another maven plugin ([/conga-aem-maven-plugi](https://devops.wcm.io/conga/plugins/aem/tooling/conga-aem-maven-plugin/plugin-info.html)). those files are generated within the target folder, but are not actually attached artifacts as they are only used for local or CM deployment, and never intended to be uploaded to an artifact manager nor to be installed in the local maven repo.

what do you want to use instead of "File"?

Reply:
and what is the harm if it is used to validate a file that is stored anywhere? every validation that is run helps improving the quality a bit...

Reply:
I have no issue with validating a file within the project directory (either source or target), but not outside as that makes a build unpredictable.
We can use "File" but then check in the code that its below the project directory

Reply:
ok, makes sense - will add a check